From 63d7f35b281dd22d1ffd3df888f5754a05e1538d Mon Sep 17 00:00:00 2001
From: Ludovic Desroches <ludovic.desroches@atmel.com>
Date: Tue, 12 Feb 2013 14:57:17 +0100
Subject: [PATCH 310/319] dt: define of_find_device_by_node when dt is not
 enabled

Temporary workaround since some at91 drivers use this function even if dt is
not enabled causing compilation issue.

Signed-off-by: Ludovic Desroches <ludovic.desroches@atmel.com>
---
 include/linux/of_platform.h |    6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/include/linux/of_platform.h b/include/linux/of_platform.h
index b47d204..25f9be1 100644
--- a/include/linux/of_platform.h
+++ b/include/linux/of_platform.h
@@ -109,4 +109,10 @@ static inline int of_platform_populate(struct device_node *root,
 }
 #endif /* !CONFIG_OF_ADDRESS */
 
+#if !defined(CONFIG_OF_DEVICE)
+static inline struct platform_device *of_find_device_by_node(struct device_node *np) {
+	return NULL;
+}
+#endif
+
 #endif	/* _LINUX_OF_PLATFORM_H */
-- 
1.7.10.4

